src/Components/Config/ConfigScreen.js

```diff
@@ -145,28 +145,30 @@ class ConfigScreen extends PureComponent {
     if (validation) {
       let isValid = true
     
-      if (!validation.canBeBlank && value == '') {
-        isValid = false
-      }
-    
-      if (validation.allowedChars && !value.match(validation.allowedChars)) {
-        isValid = false
-      }
-    
-      if (validation.format && !value.match(validation.format)) {
-        isValid = false
-      }
+      if (value !== null) {
```

Comment:
should we use strict inequality here? in an `if` statement, `null` only ever coerces to `undefined` and vice-versa...

Reply:
I'd be fine with not doing a strict inequality here as we'd want to catch `undefined` as well